index.html

```diff
@@ -32,3 +32,8 @@
 
 <!--NO JS SUPPORT-->
 <noscript><div id=area><h3>You do not have JavaScript, or it is disabled. Please enable JavaScript for the best experience. If you cannot, you may use this <a href="1mainResources/noJSmenu.html">menu</a>.</h3></div>
+<link rel="stylesheet" href="npa/mobilenpa.css" />
+<div class="invisbutton" style="color: #FFFFFF">
+  <button onclick="window.location.href='npa/LoadingNPA.htm';">Placeholder NPA button, will be invisible later</button>

```

Comment:
you could do a html button which are simpler 

Sadly I cant remember how to do them (with functions on click), but I know that there is a way

Reply:
Is that not an HTML button?